api/app/signals/apps/dataset/management/commands/load_categories.py

```diff
@@ -44,11 +45,11 @@ def handle(self, *args, **options) -> None:
                 parent_id = fields.pop('parent')
                 if parent_id:
                     fields['parent'] = self._get_parent(parent_id)
-                cat = self._get_cat(fields['slug'], fields.get('parent', None))
+                cat = self._get_cat(slugify(fields['name']), fields.get('parent', None))
```

Comment:
This will create a new category when the name changes. Is this intended?

Reply:
my point exactly :)

Reply:
this is intended :) old cat will be set to inactie